doc/read-the-docs-site/quick-start.rst

```diff
@@ -0,0 +1,127 @@
+In ``plutus-quickstart.cabal``, replace ``MyLib`` with ``AuctionValidator``.
+
+Now, build it: ::
+
+  cabal build plutus-quickstart
+
+This should succeed, and write the serialised validator to a file, whose name is similar to ``AuctionValidator.uplc147232-0.flat`` (the digits are random and yours may differ).
+Congratulations - you've successfully created a Plutus validator script.
+
+Creating and Submitting Transactions using an Off-Chain Framework
+-----------------------------------------------------------------
+
+Once you have the validator, you can proceed with deploying and interacting with a smart contract that uses this validator.
+To do so, you'll need the ability to perform operations like the following:
+
+* Generating key pairs
+* Quering available UTXOs that satisfy certain criteria and can be used as the input of a transaction
+* Building transactions and calculating transaction fees
+* Signing and submitting transactions
+
+These can be done using low-level Cardano CLI commands or the Cardano API library functions.
+A better way is to use high-level off-chain libraries and frameworks, such as:
+
+* `Lucid <https://lucid.spacebudz.io/>`_, a JavaScript off-chain library for Cardano
+* `Kuber <https://github.com/dQuadrant/kuber>`_, which provides a Haskell library and a JSON API for working with Cardano transactions
+* `cardano-transaction-lib <https://github.com/Plutonomicon/cardano-transaction-lib>`_, a PureScript library for building Cardano transactions
+
+These frameworks either consume compiled validators in serialised form (such as the one you just made), or depend on the Plutus Tx library and compile the on-chain code from source.
+Refer to their respective documentation for more details about how to use them.
+
+A good way to quickly deploy and test a smart contract is to do it on a public testnet, such as Preview.
+Generate a key pair, go to the `faucet <https://docs.cardano.org/cardano-testnet/tools/faucet/>`_ for the testnet you are using to request some funds, submit a transaction to lock the funds in your smart contract validator script, and off you go to have all the fun with it.
+Read :ref:`simple_example`, section *Life cycle of the auction smart contract*, if you need to understand how one can submit transactions to interact with the auction smart contract.
+
+Interfacing between Plutus Tx and Off-Chain Frameworks
+-----------------------------------------------------------------
+
+At this time, interfacing between Plutus Tx and most off-chain frameworks (especially non-Haskell ones) isn't very well supported.
+What this means is that you may run into inconveniences like these:
+
+* The compiled valiator obtained via ``dump-uplc`` is a ``flat`` file, but some off-chain frameworks expect a Hex string, which must be obtained by first encoding the data in the ``flat`` file using CBOR, then Hex encode the CBOR data.
```

Comment:
This is another reason not to use `dump-uplc`. Pretty sure we have a function that produces the CBOR, which is what people should always be using in transactions

Reply:
Yeah, `serialiseCompiledCode`.